packages/manager/modules/bm-server-components/src/general-information/inputs/controller.js

```diff
@@ -0,0 +1,79 @@
+import remove from 'lodash/remove';
+
+export default class BmServerComponentsInputsController {
+  /* @ngInject */
+  constructor($q, $scope, $stateParams, $translate) {
+    this.$q = $q;
+    this.$scope = $scope;
+    this.$stateParams = $stateParams;
+    this.$translate = $translate;
+  }
```

Comment:
These injections don't seem to be used, you should be able to remove this constructor

Reply:
done